src/translations/en.json

```diff
@@ -1078,6 +1078,10 @@
           },
           "editor": {
             "alias": "Name",
+            "icon": "Icon",
+            "id": "Entity ID",
+            "id_already_exists_save_error": "You can't save this script because the ID is not unique, pick another ID or leave it black to autogenerate one.",
```

Comment:
```suggestion
            "id_already_exists_save_error": "You can't save this script because the ID is not unique, pick another ID or leave it blank to automatically generate one.",
```